src/Umbraco.Web.UI.Client/src/views/documenttypes/import.html

```diff
@@ -0,0 +1,72 @@
+﻿<div class="umb-dialog umb-pane" ng-controller="Umbraco.Editors.DocumentTypes.ImportController as vm">
+    <table class="propertyPane" id="Table1" cellspacing="0" cellpadding="4" width="360" border="0" runat="server">
```

Comment:
Yes it can be removed.  This only works on aspx files.

Reply:
As I said, it's not final :)
Id & runat attributes could be dropped, yes.
Not even sure if the HTML structure is what we want, I just kept the aspx version with only the upload part being really new.
Had no more time to finish it up during the hackathon but wanted to expose the functional rework already so it could be discussed/finalized/...